### ISSUE TITLE
Plexus POM 8 provides inconsistent Checkstyle configuration that fails site

### ISSUE THREAD

**Issue by hboutemy** [codehaus-plexus/plexus-pom#103]
when trying to build maven-archiver:
```
$ mvn -Preporting site
...
[INFO] Generating "Checkstyle" report    --- maven-checkstyle-plugin:3.1.2:checkstyle
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  17.206 s
[INFO] Finished at: 2022-06-05T11:22:39+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-site-plugin:3.9.1:site (default-site) on project plexus-archiver: Error generating maven-checkstyle-plugin:3.1.2:checkstyle report: Failed during checkstyle configuration: cannot initialize module TreeWalker - cannot initialize module JavadocMethod - Property 'scope' does not exist, please check the documentation -> [Help 1]
```

**Comment by hboutemy** [comment]
https://issues.apache.org/jira/browse/MSHARED-990

**Comment by hboutemy** [comment]
fixed by #63
